tests/testthat/test-masked_unmasked.R

```diff
@@ -12,13 +13,14 @@ test_that("masked and unmasked values are processed properly", {
   df_merged_data[df_merged_data$clid == df_merged_data$clid[[3]] &
                    df_merged_data$Gnumber ==  unique(df_merged_data$Gnumber)[[3]],
                  "masked"] <- TRUE
+  df_merged_data <- data.table::setDT(df_merged_data)
```

Comment:
Replace this line by `data.table::setDT(df_merged_data)` it modifies by reference